app/models/bulk_download.rb

```diff
@@ -128,7 +128,7 @@ def s3_tar_writer_command(
   end
 
   def download_output_key
-    "downloads/#{id}/#{BulkDownloadTypesHelper::BULK_DOWNLOAD_TYPE_TO_DISPLAY_NAME[download_type]}.tar.gz"
+    "downloads/#{id}/#{BulkDownloadTypesHelper.bulk_download_type_display_name(download_type)}.tar.gz"
```

Comment:
This `id` is the id of the bulk download, right? So it is guaranteed to be unique?

Reply:
That's correct.